connect-location/src/main/java/com/ifttt/location/LocationTriggerField.java

```diff
@@ -0,0 +1,17 @@
+package com.ifttt.location;
+
+import androidx.annotation.Nullable;
+import com.ifttt.connect.LocationFieldValue;
+
+final class LocationTriggerField {
```

Comment:
We should double check the API response to finalize the data structure: we are unlikely to receive location field data in this format.